docs/content/guides/formulas/formula-calculation.md

```diff
@@ -56,9 +56,9 @@ To find out which HyperFormula version to use, see the table below:
 | [`12.x.x`](https://github.com/handsontable/handsontable/releases/tag/12.0.0) and higher | [`^2.0.0`](https://github.com/handsontable/hyperformula/releases/tag/2.0.0)                                    |
 
 ::: tip
-Your [Handsontable license key](@/guides/getting-started/license-key.md) lets you use HyperFormula for free in your Handsontable instance: just pass `'internal-use-in-handsontable'` as your HyperFormula license key .<br><br>
+You can use the `'internal-use-in-handsontable'` license key only in those HyperFormula instances that are connected to a Handsontable instance.
```

Comment:
Is `<br><br>` not needed anymore?

Reply:
No, I think it shouldn't have been there in the first place, the note looks good in the output:

<img width="668" alt="3" src="https://user-images.githubusercontent.com/29247866/209168696-703a549f-4395-42a9-b713-5370b33a6b7b.png">

Reply:
ok